.github/workflows/docker-c-latest.yml

```
@@ -0,0 +1,54 @@
+name: Publish latest Docker C extraction container
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: ['main']
+    paths: ['.docker/c/**']
+    types:
+      - closed
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}-c
+  SOURCE_TAG: unstable
+  TARGET_TAG: latest
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  setup:
+    # Only run if PR merged
+    if: ${{ github.event.pull_request.merged == true }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup tag names
+        run: echo "Setting up tag names..."
+    outputs:
+      source_image: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.SOURCE_TAG }}
+      target_image: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.TARGET_TAG }}
+  
+  
+  publish:
+    needs: [setup]
+    runs-on: ubuntu-latest
+
+    # We need write access to packages
+    permissions:
+      contents: read
+      packages: write
+      id-token: write
+    
+    steps:
+      - uses: actions/checkout@v4
+      - name: Log in to the Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Retag and push Docker image
+        id: push
+        run: docker pull ${{ needs.setup.outputs.source_image }} && docker tag ${{ needs.setup.outputs.source_image }} ${{ needs.setup.outputs.target_image }} && docker push ${{ needs.setup.outputs.target_image }}
```

This way it looks a little nice.
```suggestion
        run: |
            docker pull ${{ needs.setup.outputs.source_image }}
            docker tag ${{ needs.setup.outputs.source_image }} ${{ needs.setup.outputs.target_image }}
            docker push ${{ needs.setup.outputs.target_image }}
```